src/main/java/org/apache/hadoop/hive/ql/io/avro/AvroGenericRecordReader.java

```diff
@@ -85,22 +85,38 @@ public AvroGenericRecordReader(JobConf job, FileSplit split, Reporter reporter)
     if(latest != null) {
       gdr.setExpected(latest);
     }
+    org.apache.avro.file.FileReader<GenericRecord> nonFinalReader = null;
```

Comment:
import?

i think you can call the variable just `reader`.

Reply:
> import?
I am consitent with field declaration ;)